apis/config/v1beta1/defaults.go

```diff
@@ -23,22 +23,22 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
 
 	"sigs.k8s.io/kueue/pkg/controller/jobs/job"
 )
 
 const (
-	DefaultNamespace              = "kueue-system"
-	DefaultWebhookServiceName     = "kueue-webhook-service"
-	DefaultWebhookSecretName      = "kueue-webhook-server-cert"
-	DefaultWebhookPort            = 9443
-	DefaultHealthProbeBindAddress = ":8081"
-	DefaultMetricsBindAddress     = ":8080"
-	DefaultLeaderElectionID       = "c1f6bfd2.kueue.x-k8s.io"
-	DefaultClientConnectionQPS    = 20.0
-	DefaultClientConnectionBurst  = 30
-	defaultPodsReadyTimeout       = 5 * time.Minute
+	DefaultNamespace                      = "kueue-system"
+	DefaultWebhookServiceName             = "kueue-webhook-service"
+	DefaultWebhookSecretName              = "kueue-webhook-server-cert"
+	DefaultWebhookPort            int     = 9443
```

Comment:
```suggestion
	DefaultWebhookPort               = 9443
```

Do we need to declare explicitly `int`?